blog-posts/enterprises-benefit-from-scoped-npm-registries.md

```diff
@@ -1,8 +1,8 @@
 ---
-path: "/blog/scoped-registry/"
+path: "/blog/enterprises-benefit-from-scoped-npm-registries/"
 date: "2022-01-17"
 title: "How enterprises benefit from scoped npm registries"
-featuredImage: images/npm-scoped-registry.jpg
+featuredImage: images/enterprises-benefit-from-scoped-npm-registries.jpg
```

Comment:
images probably don't have any effect on SEO, changing the path is enough 🤓

Reply:
I know, I just wanted to clearly name it so everybody knows that it is from this blog post :)